home/templates/home/logout.html

```diff
@@ -0,0 +1,14 @@
+{% extends "home/base.html" %}
+{% block content %}
+<body class="main-bg">
+    <h1>Logged Out</h1>
+
+    <script>
```

Comment:
IIRC you can do this with an HTTP header, so you don't have to trust JS on the client side.

https://en.wikipedia.org/wiki/URL_redirection#Refresh_Meta_tag_and_HTTP_refresh_header

But it is considered a UI anti-pattern to have a flashing disappearing page like this, its better to just redirect to the main page. Users usually can tell they are logged out because their name or profile icon no longer appears on the navigation bar.

Reply:
It might be good enough to redirect immediately. It seems the LogoutView has the built-in ability to do this using the `next_page` attribute or the [`LOGOUT_REDIRECT_URL`](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-LOGOUT_REDIRECT_URL) setting.

https://docs.djangoproject.com/en/3.2/topics/auth/default/#all-authentication-views

You can also try to be a bit more clever and have the Login and Logout views redirect the users back to where they came from if possible, the can provide a better experience when the login/logout button are on a navigation bar that is seen on every page.